spec/requests/saml_sessions_controller_spec.rb

```diff
@@ -221,34 +224,74 @@
 
   def raw_details_response
     {
-      providerFirmId: 22_381,
-      contactUserId: 29_562,
-      contacts: [
-        {
-          id: 568_352,
-          name: "SALLYCORNHILL",
-        },
+      firm: {
+        firmId: 3_959_183,
+        firmNumber: "50736",
+        ccmsFirmId: 10_835_831,
+        firmName: "DT SCRIPT PROVIDER 1",
+      },
+      user: {
+        userId: 454_031,
+        ccmsContactId: 0,
+        userLogin: "DT_SCRIPT_USER1",
+        name: "Dts User1",
+        emailAddress: "deepak.tanna@digital.justice.gov.uk",
+        portalStatus: "Active",
+      },
+      officeCodes: [
```

Comment:
personally I prefer if the stubs only contain the bear minimum information to get the test to pass. This is more informative to our future selves and sets an expectation of only what we need from the API.